File: agora-easycall-ios-oc/AgoraSample/AGSSplashSegue.m
//
//  AGSSplashSegue.m
//  AgoraSample
//
//  Created by let on 10/21/15.
//  Copyright © 2015 Agora. All rights reserved.
//

#import "AGSSplashSegue.h"

@implementation AGSSplashSegue

- (void)perform
{
    [[[UIApplication sharedApplication] keyWindow] setRootViewController:self.destinationViewController];
}

@end

File: agora-easycall-ios-swift/SampleSwift/AGSSplashSegue.swift
//
//  AGSSplashSegue.swift
//  SampleSwift
//
//  Created by let on 11/2/15.
//  Copyright © 2015 Agora. All rights reserved.
//

import UIKit

class AGSSplashSegue: UIStoryboardSegue {
    
    override func perform() {
        UIApplication.sharedApplication().keyWindow?.rootViewController = destinationViewController
    }
    
}
